Cargo.lock

```diff
@@ -1506,6 +1506,12 @@ dependencies = [
  "tokio",
 ]
 
+[[package]]
+name = "maplit"
+version = "1.0.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "3e2e65a1a2e43cfcb47a895c4c8b10d1f4a61097f9f254f183aee60cad9c651d"
+
```

Comment:
Can you split these into another commit, or preferably wait until #1321 is merged and then rebase?

Reply:
#1321 is merged.